app/src/main/res/values/dimens.xml

```diff
@@ -7,4 +7,7 @@
     <dimen name="list_item_padding_horizontal">16dp</dimen>
     <dimen name="list_item_padding_vertical">10dp</dimen>
     <dimen name="list_thumbnail_size">48dp</dimen>
+
+    <dimen name="login_screen_margin">16dp</dimen>
```

Comment:
Może ujednolicimy konwencję nazewniczą paddingów/marginów? Np 16dp to spacing_normal, 10 dp spacing_small? Co o tym sądisz?